.containerignore

```diff
@@ -0,0 +1,2 @@
+# ignores .bonfire_venv from https://github.com/RedHatInsights/cicd-tools/blob/main/bootstrap.sh#L76
```

Comment:
Since there are some builds with `docker`, are we sure that we want to use `.containerignore` instead of `.dockerignore`?

Reply:
added both